camel-core/src/main/java/org/apache/camel/model/cloud/ServiceCallExpressionConfiguration.java

```diff
@@ -248,13 +248,13 @@ public Expression newInstance(CamelContext camelContext) throws Exception {
 
                     parameters.replaceAll(
                         (k, v) -> {
-                            if (v != null && v instanceof String) {
+                            if (v instanceof String) {
                                 try {
                                     v = camelContext.resolvePropertyPlaceholders((String) v);
                                 } catch (Exception e) {
                                     throw new IllegalArgumentException(
-                                        String.format("Exception while resolving %s (%s)", k, v.toString()),
-                                        e
+                                            String.format("Exception while resolving %s (%s)", k, v.toString()),
+                                            e
```

Comment:
However, I would remove this change, or explain why it is needed.

Reply:
Will remove spurious formatting, note this is because Camel no longer provides intellij IDE configs mentioned here: http://camel.apache.org/set-up-your-ide.html
I'm completely liable (especially when using refactoring utils) to change details like this without being aware